### ISSUE TITLE
[CommandBar] Toolbar buttons are not visible while using High Contrast #1 and High Contrast Black themes

### ISSUE THREAD

**Issue by codehag** [firefox-devtools/debugger#4181]
![](https://bugzilla.mozilla.org/attachment.cgi?id=8905832)
[Affected versions]:
Firefox 57.0a1 (BuildId:20170908220306)
Firefox 56.0b9 (BuildId:20170903140023)

[Affected platforms]:
Windows 7 64bit.

[Steps to reproduce]:
1. Launch Firefox.
2. Change Windows display to High Contrast #1 or High Contrast Black.
3. Access the https://devtools-html.github.io/debugger-examples/examples/todomvc/ webpage.
4. Open the debugger.

[Expected result]:
All the debuggers elements are properly displayed and no UI issues occur.

[Actual result]:
Without setting and pausing at a breakpoint:

 -The "Pause" button is visible only at hover.

After setting and pausing at a breakpoint:

 -All buttons (except the "Ignore exceptions" button) are visible only at hover.

[Regression range]:
I don't think that this is a regression.

[Additional Information]
Please observe the attached screencast for further information regarding this issue.

**Comment by codehag** [comment]
Linked Bugzilla issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1398064

**Comment by Triscy** [comment]
/claim

Claiming this for Fire Ferrets, CS 490.

**Comment by jasonLaster** [comment]
Ping @Triscy how's it going?

**Comment by Triscy** [comment]
@jasonLaster Moving along slowly. Planning stages alongside coursework.

**Comment by Triscy** [comment]
Meant to comment on this earlier; I can confirm that this is also an issue on Windows 10 64-bit, version 56.0.

**Comment by Triscy** [comment]
/unclaim

We believe that there may be an issue in the svg files: resume.svg, pause.svg, pause-exceptions.svg, stepIn.svg, stepOut.svg, and stepOver.svg, used by the CommandBar.js and CommandBar.css files. The current files are missing a fill attribute which we believe should be in the files to establish a definitive color which may be getting changed at some point in the current code. Another problem is that the file resume.svg is missing the fill-rule that all the other svg files have. The last possible cause that we have thought of is that there may be some way that the Windows operating system is interacting with the colors when it changes to high contrast that may be messing with how they get displayed, but we were not able to dig too deep into that theory.

We ran out of time to work on it, and none of us have the time to continue on this bug. Hopefully what we figured out will be helpful to whoever picks this up next. Good luck, friends, and take care! 

From
The Fire Ferrets

**Comment by sonicScape211** [comment]
Hey guys! So my team and I have taken a look into this issue more and have found that the problem continues throughout the Firefox versions: 56.0 to 58.0 but before and after these versions the problem is no longer present. Though my team and I are no longer about to keep working on this one just because of the need get Windows 7 to recreate the bug at all.

**Comment by yogmel** [comment]
Hi everyone,
I was just passing by and tried to test on my Windows 7 machine. It seems to be working fine!
This is the screenshot from Firefox 65.0, on High Contrast #1
![image](https://user-images.githubusercontent.com/3901809/53777349-d4c3c000-3ed7-11e9-8997-1dc9c6f00e2f.png)

**Comment by janodvarko** [comment]
Firefox Developer Tools (including Debugger tool) development is now entirely happening in Bugzilla and new issues should be reported here: https://bugzilla.mozilla.org/enter_bug.cgi?product=DevTools&component=Debugger

Please reopen this report in Bugzilla if it's still valid.

Honza
